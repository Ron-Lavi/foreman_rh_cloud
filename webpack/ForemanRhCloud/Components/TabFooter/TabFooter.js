import React from 'react';
import PropTypes from 'prop-types';
import { Grid } from 'patternfly-react';

const TabFooter = ({ children }) => (
  <Grid.Row>
    <div className="tab-footer">{children}</div>
  </Grid.Row>
);

TabFooter.propTypes = {
  children: PropTypes.node,
};

TabFooter.defaultProps = {
  children: null,
};

export default TabFooter;
