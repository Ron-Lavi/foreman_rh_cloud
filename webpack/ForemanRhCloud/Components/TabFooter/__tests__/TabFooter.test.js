import { testComponentSnapshotsWithFixtures } from '@theforeman/test';

import TabFooter from '../TabFooter';

const fixtures = {
  'render without Props': {},
  /** fixtures, props for the component */
};

describe('TabFooter', () => {
  describe('rendering', () =>
    testComponentSnapshotsWithFixtures(TabFooter, fixtures));
});
