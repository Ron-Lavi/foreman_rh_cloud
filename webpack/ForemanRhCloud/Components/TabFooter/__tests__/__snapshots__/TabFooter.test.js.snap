// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabFooter rendering render without Props 1`] = `
<Row
  bsClass="row"
  componentClass="div"
>
  <div
    className="tab-footer"
  />
</Row>
`;
